main.js

```diff
@@ -248,15 +253,22 @@ function join() {
 						let totalWaitTime = getWaitTime(queueStartPlace, 0);
 						let timepassed = getWaitTime(queueStartPlace, positioninqueue);
 						let ETAmin = (totalWaitTime - timepassed) / 60;
-						server.motd = `Place in queue: ${webserver.queuePlace} ETA: ${webserver.ETA}`; // set the MOTD because why not
+
+                                                var req = new XMLHttpRequest();
```

Comment:
why not use minecraft-protocol for this request?

Reply:
No reason other than unfamiliarity with said package

Reply:
Definelty minecrraft-protcol will be cool.